app/views/catalog_manager/shared/_available_statuses.html.haml

```diff
@@ -17,27 +17,24 @@
 -# DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
 -# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
 -# TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
 #all_statuses{style: @organization.use_default_statuses ? "display: none;" : "display: block;"}
```

Comment:
I noticed during manual testing that this page makes a large number of cache hits. We may be able to use a combination of eager loading and Ruby enumerable selection to improve performance.

Reply:
The large number of cache hits is due to how we retrieve statuses from permissible values. Stuart informed me that we're going to change how we initialize variables from permissible values, so the issue will be solved when that is complete.